zuul.d/project-templates.yaml

```diff
@@ -53,33 +53,56 @@
     name: ansible-collections-amazon-aws
     check:
       jobs:
+        - ansible-test-sanity-docker
+        - ansible-test-sanity-docker-stable-2.9
+        - ansible-test-sanity-docker-stable-2.10
+        - ansible-test-units-amazon-aws-python38
         - build-ansible-collection:
             required-projects:
               - name: github.com/ansible-collections/ansible.netcommon
               - name: github.com/ansible-collections/ansible.utils
+        - ansible-test-cloud-integration-aws-py36:
+            voting: false
     gate:
       jobs:
+        - ansible-test-sanity-docker
+        - ansible-test-sanity-docker-stable-2.9
+        - ansible-test-sanity-docker-stable-2.10
+        - ansible-test-units-amazon-aws-python38
         - build-ansible-collection:
             required-projects:
               - name: github.com/ansible-collections/ansible.netcommon
               - name: github.com/ansible-collections/ansible.utils
-
+        - ansible-test-cloud-integration-aws-py36:
+            voting: false
 - project-template:
     name: ansible-collections-community-aws
     check:
       jobs:
+        - ansible-test-sanity-docker
+        - ansible-test-sanity-docker-stable-2.9
+        - ansible-test-sanity-docker-stable-2.10
+        - ansible-test-units-community-aws-python38
         - build-ansible-collection:
             required-projects:
               - name: github.com/ansible-collections/amazon.aws
               - name: github.com/ansible-collections/ansible.netcommon
               - name: github.com/ansible-collections/ansible.utils
+        - ansible-test-cloud-integration-aws-py36:
+            voting: false
     gate:
       jobs:
+        - ansible-test-sanity-docker
+        - ansible-test-sanity-docker-stable-2.9
+        - ansible-test-sanity-docker-stable-2.10
+        - ansible-test-units-community-aws-python38
         - build-ansible-collection:
             required-projects:
               - name: github.com/ansible-collections/amazon.aws
               - name: github.com/ansible-collections/ansible.netcommon
               - name: github.com/ansible-collections/ansible.utils
+        - ansible-test-cloud-integration-aws-py36:
+            voting: false
```

Comment:
please remove, no need for non-voting jobs in gate. Basically a waste of resources

Reply:
it looks like this got reverted. We should remove this in gate.